Amazon Scraper/girlScriptAmazonScrape/items.py

```diff
@@ -0,0 +1,15 @@
+# Define here the models for your scraped items
+#
+# See documentation in:
+# https://docs.scrapy.org/en/latest/topics/items.html
+
+import scrapy
+
+
+class GirlscriptamazonscrapeItem(scrapy.Item):
```

Comment:
The script has nothing to do with GirlScript except the fact that you're contributing via this program. Therefore, name the class as `AmazonScrap` and replace it in all the instances in other files too. Also, edit the name of the subfolder too, `girlScriptAmazonScrape` to `Scrap`. If any path changes are required in the program then do that also